scripts/tasks/unsafe-external-contributor-ci-workaround.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+
+# CI fails for external contributors because they do not have access to the secrets necessary to run
+# all jobs (namely, Sauce Labs). This script is a workaround to copy the branch from the external
+# contributor's fork and push it to the main repo. This causes CI to be executed by the internal
+# maintainer, and therefore have full access to all the secrets.
+# WARNING: This is not safe! Review all code changes in the PR before granting access to secrets.
+#
+# Usage: yarn external-contributor-ci-workaround <PR number> [remote=origin]
```

Comment:
I don't think anyone has the remote named anything other than `origin`, so I would say YAGNI and remove this option.

Reply:
I might not need it, but what if I _want_ it? 😝 

Given the relative unimportance of this particular code, I'm gonna leave it for the sake of being a completionist.